front/app/modules/commercial/content_builder/services/ContentBuilder.ts

```diff
@@ -0,0 +1,48 @@
+import { API_PATH } from 'containers/App/constants';
+import streams from 'utils/streams';
+
+export interface IBuilderLayoutData {
+  type: 'project';
+  id: string;
+  attributes: {
+    craftjs_jsonmultiloc: JSON;
+    code: string;
+    enabled: boolean;
+    created_at: string;
+    updated_at: string;
+  };
+}
+
+export interface IBuilderLayout {
+  data: IBuilderLayoutData;
+}
+
+export function builderLayoutStream(projectId: string, code: string) {
+  return streams.get<IBuilderLayout>({
+    apiEndpoint: `${API_PATH}/projects/${projectId}/content_builder_layouts/${code}`,
+  });
+}
+
+export function updateBuilderLayout(
+  projectId: string,
+  code: string,
+  dataId: string,
+  object: IBuilderLayout
+) {
+  return streams.update<IBuilderLayout>(
+    `${API_PATH}/projects/${projectId}/content_builder_layouts/${code}/upsert`,
+    dataId,
+    { layout: object }
+  );
+}
+
+export function deleteBuilderLayout(
```

Comment:
No need for a delete service - we won't need it for now so we can safely remove it :)

Reply:
@IvaKop Ok! I noticed it's been implemented in the backend (at least, in the specification it mentions an endpoint for: "DELETE projects/:project_id/content_builder_layouts/:code  To delete a layout"). I guess I'm curious why we would implement it in the backend but not the front for this iteration?